### PR TITLE
fixes to help msgs for injection group

### PR DESCRIPTION
**Pull request by tdent**
Note that the 'injection-scale-factor' option is advertised as both 'multiply injections by this factor' and 'divide injections by this factor' in the single-detector / multi-detector interfaces.  This has not been changed in this PR so far, but it is highly suspicious.

The two descriptions should not both be correct.  So which one ought to be changed?

### CODE REVIEW

File: pycbc/strain/strain.py [hunk 1]
```diff
@@ -508,29 +508,35 @@ def insert_strain_option_group(parser, gps_times=True):
...
+                           "used")
+    data_reading_group.add_argument("--injection-f-ref", type=float,
+                      help="Reference frequency in Hz for creating CBC "
+                           "injections from an XML file")
+    data_reading_group.add_argument("--injection-f-final", type=float,
+                      help="Sample rate to use for injections (integer Hz). "
+                           "Typically similar to the strain data sample rate."
+                           "If not provided, the strain sample rate will be "
+                           "used")ide the f_final field of a CBC XML "
+                           "injection file (frequency in Hz)")
```
Comment (titodalcanton) [inline pycbc/strain/strain.py:533]: Huh?

**Comment by spxiwh** [comment]
@tdent This scale-factor is passed directly here as `distance_scale`:

https://github.com/gwastro/pycbc/blob/master/pycbc/inject/inject.py#L96

so it's dividing the injections by this factor. (For e.g. see the "no injection made" settings in the config files, where we set this to something large to mimic "no injection made" because that's easier than actually turning the injection off!)

File: pycbc/strain/strain.py (hunk 1 above)
Comment (tdent) [reply pycbc/strain/strain.py:530]: yes, the tests didn't like this copy paste typo either

**Comment by tdent** [comment]
OK, so it sounds like the help message for the multi inspiral injection scale factor is wrong. (Which is preferable to the one we've been using in production code for >1year being wrong)

**Review by spxiwh — APPROVED** [review]
No code changes so no risk of breaking anything.

Updated help messages make them accurate now, which is good!

**Comment by tdent** [comment]
I'm getting test errors like `RuntimeError: Could not find a RFC 3820 compliant proxy credential. Please run 'grid-proxy-init -rfc' and try again.` 

is any action required on that?

**Comment by spxiwh** [comment]
I think the issue is GWOSC not being responsive:
```
Failed on attempt 1 to download https://www.gw-openscience.org/timeline/segments/json/O1/H1_DATA/1126051217/100000/
Failed on attempt 2 to download https://www.gw-openscience.org/timeline/segments/json/O1/H1_DATA/1126051217/100000/
Failed on attempt 3 to download https://www.gw-openscience.org/timeline/segments/json/O1/H1_DATA/1126051217/100000/
Failed on attempt 4 to download https://www.gw-openscience.org/timeline/segments/json/O1/H1_DATA/1126051217/100000/
Failed on attempt 5 to download https://www.gw-openscience.org/timeline/segments/json/O1/H1_DATA/1126051217/100000/
Giving up on https://www.gw-openscience.org/timeline/segments/json/O1/H1_DATA/1126051217/100000/
Could not query flag, check name (DATA) or times
```

**Comment by tdent** [comment]
Could be, so maybe stuff will work tomorrow ..

**Pull request merged**